encrypted-media-respec.html

```diff
@@ -3172,7 +3213,7 @@
 
         <section id="allow-identifiers-cleared">
           <h3>Allow Identifiers to Be Cleared</h3>
-          <p>As a consequence of the requirements in <a href="#allow-persistent-data-cleared">Allow Persistent Data to Be Cleared</a>, implementations that use <a def-id="distinctive-identifier">Distinctive Identifier(s)</a> MUST allow the user to clear those identifiers such that they are no longer retrievable both outside, such as via the APIs defined in this specification, and on the client device.
+          <p>As a consequence of the requirements in <a href="#allow-persistent-data-cleared">Allow Persistent Data to Be Cleared</a>, implementations that <a href="#uses-distinctive-identifiers-or-distinctive-permanent-identifiers">Distinctive Identifier(s) or Distinctive Permanent Identifier(s)</a> MUST allow the user to clear the <a def-id="distinctive-identifier">Distinctive Identifier(s)</a> and values associated with <a def-id="distinctive-permanent-identifier-maybe-plural"></a> such that they are no longer retrievable both outside, such as via the APIs defined in this specification, and on the client device.
```

Comment:
I see what you are trying to say. I think the problem is that I believe Distinctive Permanent Identifiers are Distinctive Identifiers, at least according to the definitions.

This might be a bigger problem than just here, since we place normative requirements on Distinctive Identifiers which Distinctive Permanent Identifiers cannot meet. Or do we place those requirements only on Distinctive Identifiers that are exposed to applications ?

If the latter, you could replace:
> allow the user to clear the Distinctive Identifier(s)
with
> allow the user to clear all Distinctive Identifier(s) that are not Distinctive Permanent Identifier(s)